src/ui/components/SecondaryToolbox/ReactDevTools.tsx

```diff
@@ -260,7 +257,7 @@ export function ReactDevtoolsPanel() {
       // Inject the RDT backend and prefetch node IDs
       wall.setUpRDTInternalsForCurrentPause();
     }
-  }, [currentPoint, wall]);
+  }, [currentPoint, currentTime, wall]);
```

Comment:
This change seems unnecessary/mistaken.
```suggestion
  }, [currentPoint, wall]);
```